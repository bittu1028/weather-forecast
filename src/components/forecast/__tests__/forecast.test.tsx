// __tests__/searchBox.test.js
import React from 'react'
import { render, screen } from '@testing-library/react'
import '@testing-library/jest-dom'
import { mockForecastData } from '../../../test/mockData'
import Forecast from '../Forecast'

describe('Forecast', () => {
  test('renders current forecast component', async () => {
    const { asFragment } = render(<Forecast forecast={mockForecastData} />)
    expect(asFragment).toMatchSnapshot()
  })

  test('does not render first  item', async () => {
    render(<Forecast forecast={mockForecastData} />)
    const text = screen.queryByText('20.94')
    expect(text).not.toBeInTheDocument()
  })
})
