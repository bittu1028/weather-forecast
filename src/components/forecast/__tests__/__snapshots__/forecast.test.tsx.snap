// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Forecast renders current forecast component 1`] = `[Function]`;
